cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,23 @@
+import time
+
+from django.core.management import BaseCommand
+from django.db import connections
+from psycopg2 import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for db to start", ending="\n")
+        db = None
+        attempts = 5
+        while not db and attempts > 0:
+            try:
+                db = connections["default"]
+                self.stdout.write(self.style.SUCCESS("Connected to db"))
+            except OperationalError:
+                attempts -= 1
+                self.stdout.write("DB is unavailable, try again in 1 second..")
```

Comment:
The message 'DB is unavailable, try again in 1 second..' could be improved by including the number of remaining attempts to give more context to the user.